providers/cassandra/shedlock-provider-cassandra/src/main/java/net/javacrumbs/shedlock/provider/cassandra/CassandraStorageAccessor.java

```diff
@@ -185,6 +187,6 @@ private boolean updateUntil(String name, Instant until) {
 
 
     private boolean execute(SimpleStatement statement) {
-        return cqlSession.execute(statement.setConsistencyLevel(consistencyLevel)).wasApplied();
+        return cqlSession.execute(statement.setConsistencyLevel(consistencyLevel).setSerialConsistencyLevel(serialConsistencyLevel)).wasApplied();
```

Comment:
Don;t we need to set it also in `find` method?

Reply:
No, we don't require in `find` method because that is a normal query and not a CAS operation.